ext/exif/exif.stub.php

```diff
@@ -4,15 +4,15 @@
 
 function exif_tagname(int $index): string|false {}
 
-/** @param resource|string $filename */
-function exif_read_data($filename, ?string $sections_needed = null, bool $sub_arrays = false, bool $read_thumbnail = false): array|false {}
+/** @param resource|string $file */
+function exif_read_data($file, ?string $sections = null, bool $as_arrays = false, bool $read_thumbnail = false): array|false {}
```

Comment:
I'm uncertain about $sections vs $sections_needed/$needed_sections. Based on just $sections I would probably assume that it means "return only these sections", while it's really "return all sections if at least one of these is present".

Reply:
Hmm.. `$required_sections`? :) I feel that that's the best term

Reply:
Yes, `$required_sections` is great :)